README.md

```diff
@@ -1 +1,366 @@
+| OP_DATASOURCE_PW | jdbc connection pw (without quotes). For more information check out the regular or wallet database section below |  |
+| OP_SERVER_ENVIRONMENT | Environment mode "prod" has by default disabled Swagger and PasscodeController (without quotes) | "dev","qa","prod" |
+| OP_USER_VALIDATION_METHOD | user is not a robot checking (just number) | 0 - always true, 1- google captcha + honeypot , 2-honeypot |
+| OP_CAPTCHA_SERVER_KEY | Google captcha server key string if enabled (without quotes) |  |
+| OP_JWT_TOKEN_SIGN_KEY | signing key for jwt token (without quotes) |  |
+| OP_JWT_TOKEN_EXPIRATION_TIME_IN_MINUTES | JWT token expiration time (just number) |  |
+| OP_REGISTRATION_START_TIME | timestamp of the start date without seconds | 2019.06.08 00:00 |
+| OP_LOG_FILE_PATH | Logfile path | /var/log |
+| OP_CORS_ALLOWED_ORIGINS | Comma separated list of domains that can query the backend API | "https://opendays-registration.cern.ch" for production. By default the variable is set in the application.properties to http://localhost:4200,https://opendays-registration-demo.cern.ch|
+| OP_CORS_PATH_PATTERN | Allowed origin URL path. This path will be the same for all dev, test, prod profiles | "/**" In application.properties this value is already set so this environment variable is OPTIONAL|
+| OP_WALLET_CONNECTION | If the connection between the application and the database is via wallet. For more information please read the wallet database connection below | true,false  |
+
+For reCAPTCHA the solution supports the google reCAPTCHA v2. In order to use the reCAPTCHA you will need your own recaptcha site configuration and include the server key in the ./Dockerfile or in the ./application/main/resources/application.properties
+
+**regular database connection**
+
+Note that the containerize/extra-jars folder contains the Oracle specific connection jars. In case you need other jar(s) for database connection you have to include that in the Dockerfile.
+
+If you want to go with the regular connection you need the following settings at build time:
+* update the following connection string according to your database settings
+"jdbc:oracle:thin:@//service:port/server_name" (service name is the name of the service,port the database connection port, server name is the server name where the database is located.) and set the "OP_DATASOURCE_URL" to that
+* set the database credentials for "OP_DATASOURCE_USERNAME" and "OP_DATASOURCE_PW"
+* set the "OP_WALLET_CONNECTION" to "false"
+
+**wallet database connection**
+
+Note that the containerize/extra-jars folder contains the Oracle specific connection jars. In case you need other jar(s) for database connection you have to include that in the Dockerfile.
+
+Some databases for example the Oracle Autonomous database requires a database wallet for the connection. The wallet is a zip file containing the configurations for SSL connection. 
```

Comment:
database wallet: perhaps it would deserve a link to Oracle doc, or a "how to" article. Not everybody is familiar with the concept

Reply:
We discussed, we don't add link , user can google it.